src/main/java/org/apache/maven/shared/utils/xml/Xpp3DomBuilder.java

```diff
@@ -84,14 +84,14 @@ public static Xpp3Dom build( @WillClose InputStream is, @Nonnull String encoding
         }
         catch ( UnsupportedEncodingException e )
         {
-            throw new RuntimeException( e );
+            throw new XmlPullParserException( e );
         }
     }
 
     /**
      * @param reader {@link Reader}
```

Comment:
There is no such parameter `reader`, as in #58 it was renamed to `in`.

Reply:
fixed